source/vibe/inet/url.d

```diff
@@ -242,7 +262,14 @@ struct URL {
 			dst.put(password);
 			dst.put('@');
 		}
+
+		import std.algorithm.searching : canFind;
```

Comment:
Just noticed that this causes the Travis-CI build to fail. Needs to be `import std.algorithm : canFind` for now.